compiler/rustc_session/src/options.rs

```diff
@@ -1585,6 +1585,8 @@ options! {
         "choose the TLS model to use (`rustc --print tls-models` for details)"),
     trace_macros: bool = (false, parse_bool, [UNTRACKED],
         "for every macro invocation, print its name and arguments (default: no)"),
+    track_diagnostics: bool = (false, parse_bool, [TRACKED],
```

Comment:
This doesn't need to be a tracked flag, does it? It just affects runtime behavior I thought.

Reply:
yeah this can be untracked